src/components/data/LocalVariablesContent.js

```diff
@@ -80,17 +109,26 @@ const LocalVariablesContent = ({ workspace, workspace: { workspace: { googleProj
   )(async originalKey => {
     const isList = editType.includes('list')
     const newBaseType = isList ? editType.slice(0, -5) : editType
-
     const parsedValue = isList ? _.map(Utils.convertValue(newBaseType), editValue.split(/,\s*/)) :
       Utils.convertValue(newBaseType, editValue)
 
-    await Ajax().Workspaces.workspace(namespace, name).shallowMergeNewAttributes({ [editKey]: parsedValue })
+    const editDescriptionKey = toDescriptionKey(editKey)
 
-    if (editKey !== originalKey) {
-      await Ajax().Workspaces.workspace(namespace, name).deleteAttributes([originalKey])
+    const attributesToMerge = {
+      [editKey]: parsedValue,
+      ...(!!editDescription && { [editDescriptionKey]: editDescription })
     }
 
+    const attributesToDelete = _.compact(_.flatten([
+      editKey !== originalKey && [originalKey, toDescriptionKey(originalKey)],
+      !editDescription && [editDescriptionKey]
+    ]
+    ))
+
+    await Ajax().Workspaces.workspace(namespace, name).deleteAttributes(attributesToDelete)
+    await Ajax().Workspaces.workspace(namespace, name).shallowMergeNewAttributes(attributesToMerge)
```

Comment:
Could we flip these, as they were before, so we don't try to delete until we've created?
```suggestion
    await Ajax().Workspaces.workspace(namespace, name).shallowMergeNewAttributes(attributesToMerge)
    await Ajax().Workspaces.workspace(namespace, name).deleteAttributes(attributesToDelete)
```

Reply:
Done.